### PR TITLE
feat: add alertmanager rock

### PR DESCRIPTION
**Pull request by lucabello**
- [x] I have signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [x] I am following the [CONTRIBUTING](https://github.com/canonical/oci-factory/blob/main/CONTRIBUTING.md) guidelines

*Ping the @canonical/rocks team.*

---

## Description
This PR adds the Alertmanager ROCK to the oci-factory.

Please note that this should go to a new `alertmanager` repository that's yet to be created. There currently exists a [prometheus-alertmanager](https://hub.docker.com/r/ubuntu/prometheus-alertmanager) repository, but that one should be deprecated.

### Related issues
#42

### CODE REVIEW

File: oci/alertmanager/documentation.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+version: 1
+# --- OVERVIEW INFORMATION ---
+application: alertmanager
+description: >
+  The Alertmanager handles alerts sent by client application such as the
+  Prometheus server. It takes care of deduplicating, grouping, and routing them
+  to the correct receiver integrations such as email, PagerDuty, OpSGenie, or
+  many other mechanisms thanks to the webhook receiver. It also takes care of
+  silencing and inhibition of alerts.
+  Read more on the [official documentation](https://prometheus.io/docs/alerting/latest/alertmanager/)
```
Comment (cjdcordeiro) [inline oci/alertmanager/documentation.yaml:10]:
```suggestion
  The Alertmanager handles alerts sent by client applications such as the
  Prometheus server. It takes care of deduplicating, grouping, and routing them
  to the correct receiver integrations such as email, PagerDuty, OpSGenie, or
  many other mechanisms thanks to the webhook receiver. It also takes care of
  silencing and inhibition of alerts.
  Read more on the [official documentation](https://prometheus.io/docs/alerting/latest/alertmanager/)
```

**Review by cjdcordeiro — CHANGES_REQUESTED** [review]
Keep them coming :D

Just a small typo to fix, but it LGTM ;)

**Comment by lucabello** [comment]
Fixed, thanks!

**Comment by cjdcordeiro** [comment]
@lucabello this image also failed the efficiency and vulnerability tests-> https://github.com/canonical/oci-factory/actions/runs/5940848690/job/16110358625

**Pull request merged**